code/datums/supply_packs/clothing.dm

```diff
@@ -1,110 +1,7 @@
-//---------------------------------------------
-//POUCHES BELOW THIS LINE
-//---------------------------------------------
-/datum/supply_packs/pouches_general
-	name = "general pouches crate (2x normal, 1x medium, 1x large)"
-	contains = list(
-					/obj/item/storage/pouch/general,
-					/obj/item/storage/pouch/general,
-					/obj/item/storage/pouch/general/medium,
-					/obj/item/storage/pouch/general/large
-					)
-	cost = RO_PRICE_VERY_CHEAP
-	containertype = /obj/structure/closet/crate
-	containername = "general pouches crate"
-	group = "Clothing"
-
-/datum/supply_packs/pouches_weapons
-	name = "weapons pouches crate (1x bayonet, pistol, explosive)"
-	contains = list(
-					/obj/item/storage/pouch/bayonet,
-					/obj/item/storage/pouch/pistol,
-					/obj/item/storage/pouch/explosive
-					)
-	cost = RO_PRICE_VERY_CHEAP
-	containertype = /obj/structure/closet/crate
-	containername = "weapons pouches crate"
-	group = "Clothing"
-
-/datum/supply_packs/pouches_ammo
-	name = "ammo pouches crate (1x normal, large, pistol, pistol large)"
-	contains = list(
-					/obj/item/storage/pouch/magazine,
-					/obj/item/storage/pouch/magazine/large,
-					/obj/item/storage/pouch/magazine/pistol,
-					/obj/item/storage/pouch/magazine/pistol/large
-					)
-	cost = RO_PRICE_VERY_CHEAP
-	containertype = /obj/structure/closet/crate
-	containername = "ammo pouches crate"
-	group = "Clothing"
-
-/datum/supply_packs/pouches_medical
```

Comment:
should keep this, autoinjector pouches are limited

Reply:
done